app/controllers/api/v2/interfaces_controller.rb

```diff
@@ -6,6 +6,8 @@ class InterfacesController < V2::BaseController
       include Api::Version2
       include Api::TaxonomyScope
 
+      LEGACY_TYPE_NAMES = Nic::Base.allowed_types.map{ |t| t.name }
```

Comment:
This is never used here? It seems like it was copied from interface_type_mapper but here it's not used.